apps/explorer/src/ui/SenderRecipient.tsx

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva } from 'class-variance-authority';
+
+//TODO: Switch to using ButtonOrLink component after support for sui explorer links
+import Longtext from '~/components/longtext/Longtext';
```

Comment:
Don't use non-UI components in the UI components.

Reply:
I will update the `ButtonOrLink` account for the explorer links

Reply:
We also have a `Link` component for visually-presented links, or you could reasonably make an `AddressLink` component.